test/e2e/apps/deployment.go

```diff
@@ -25,6 +25,7 @@ import (
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 
+	apps "k8s.io/api/apps/v1beta2"
```

Comment:
do these e2e tests run against old clusters? if so, do we need to gate this test to only run on 1.8 and newer clusters?